src/curve/constant-product/ConstantProductPair.sol

```diff
@@ -87,23 +89,23 @@ contract ConstantProductPair is ReservoirPair {
         // ASSERT: newK & oldK        < uint104
         // ASSERT: aPlatformFee       < FEE_ACCURACY
         // ASSERT: aCirculatingShares < uint104
+        unchecked {
```

Comment:
We added unchecked here because it was in the original formula?

Reply:
we added an unchecked here cuz your comments (perf) said it can be unchecked.

So I made an assessment and concluded that it is indeed safe to make it unchecked